deploy/charts/mixer/templates/serviceaccount.yaml

```diff
@@ -0,0 +1,11 @@
+{{- if .Values.serviceAccount.create -}}
```

Comment:
nit: is it possible to add the license header?

Reply:
Done.